DependencyInjection/DoctrineMongoDBExtension.php

```diff
@@ -534,7 +537,12 @@ protected function getMappingResourceExtension(): string
 
     protected function getMetadataDriverClass(string $driverType): string
     {
-        return '%' . $this->getObjectManagerElementName('metadata.' . $driverType . '.class') . '%';
+        return match ($driverType) {
+            'driver_chain' => MappingDriverChain::class,
```

Comment:
We don't have any consts for these values I presume?

Reply:
Or maybe this should become an enum while we're here? :)

Reply:
The `getMetadataDriverClass` method is defined and used in the parent class, from doctrine bridge. That's something that needs to be done in the bridge first.